core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -1167,8 +1168,11 @@ trait EntityComponent {
       } yield numEntitiesHidden.sum
 
     // perform actual deletion (not hiding) of all entities in a workspace
-
     def deleteFromDb(workspaceContext: Workspace): WriteAction[Int] =
+      filter(_.workspaceId === workspaceContext.workspaceIdAsUUID).delete
+
+    @VisibleForTesting
+    def deleteEntitiesAndAttributesFromDb(workspaceContext: Workspace): WriteAction[Int] =
       EntityDependenciesDeletionQuery.deleteAction(workspaceContext) andThen
         filter(_.workspaceId === workspaceContext.workspaceIdAsUUID).delete
```

Comment:
This keeps unit tests passing. We should/will fully delete all code that relies on those tables, but that's out of scope of this PR.